althea_kernel_interface/src/ip_route.rs

```diff
@@ -6,15 +6,14 @@ use failure::Error;
 
 impl KernelInterface {
     fn get_default_route(&self) -> Option<Vec<String>> {
-        let output = self
-            .run_command("ip", &["route", "list", "default"])
+        let output = self.run_command("ip", &["route", "list", "default"])
             .unwrap();
 
         let stdout = String::from_utf8(output.stdout).unwrap();
 
         let mut def_route = Vec::new();
 
-        let mut found = false;
+        //let mut found = false;
```

Comment:
I like this change you can just delete this line though, no need to leave it commented out.